build/dockerfiles/linux-libc.Dockerfile

```diff
@@ -47,8 +47,7 @@ RUN { if [[ $(uname -m) == "s390x" ]]; then LIBSECRET="\
     fi; } \
     && yum install -y $LIBSECRET $LIBKEYBOARD curl make cmake gcc gcc-c++ python3.9 git git-core-doc openssh less libX11-devel libxkbcommon bash tar gzip rsync patch \
     && yum -y clean all && rm -rf /var/cache/yum \
-    && npm install -g yarn@1.22.17 \
-    && npm install -g node-gyp@9.4.1
```

Comment:
it looks like removing this change leads to the original problem...
I mean  - just adding the dependency to the package.json doesn't fix the problem

Reply:
I've put it back ... let's see if the build passes.